### PR TITLE
refactor: realtime log entry revamp

### PR DESCRIPTION
**Pull request by JoelJacobStephen**
### Description

Revamped the log component and included a new component that handles the UI for each log entry. Changed type definition for log entry data (removed color attribute, added event attribute and refactored the timestamp attribute).

### Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed

### CODE REVIEW

**Comment by liyasthomas** [comment]
- Copy button on the log entry is broken, clicking a copy button makes it invisible.
- The entire hoer logic should be removed from the script setup. Instead, use plain simple `CSS` / WindiCSS classes. Pretty sure there are classes like `group`, `group-hover` etc to cover these use cases.
- `SmartTab` logic inside the expanded section seems wrong. 

If we're merging this to `main`, this PR should accommodate the following features as well:
- [ ] Toggle button to enable auto-scroll to the last entry.
- [ ] Button to delete all log entries
- [ ] Virtual scroller [optional]

File: packages/hoppscotch-app/assets/icons/arrow-down-left.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="#eaaa45" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
```
Comment (liyasthomas) [inline packages/hoppscotch-app/assets/icons/arrow-down-left.svg:1]: Icons should always have `currentColor`. Do not give fixed colors, instead use a computed class name to color the icons where they're being used.

File: packages/hoppscotch-app/assets/icons/arrow-up-right.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="#38d4ea" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
```
Comment (liyasthomas) [inline packages/hoppscotch-app/assets/icons/arrow-up-right.svg:1]: All Icons* should use `currentColor`.

**Review by liyasthomas — CHANGES_REQUESTED** [review]
@JoelJacobStephen, cc: @AndrewBastin
- [x] Fix broken "delete logs button" on the `SSE`, `Socket.IO`, and `MQTT` sections.
- [x] Remove fixed colors from svg icons

**Comment by AndrewBastin** [comment]
Looks fine in my end.

@liyasthomas please review and merge.

**Review by liyasthomas — APPROVED** [review]
LGTM

**Pull request merged**